meson.build

```diff
@@ -601,11 +601,17 @@ You can disable the Wayland xkbcli programs with -Denable-wayland=false.''')
             wayland_scanner_code_gen.process(xdg_shell_xml),
             wayland_scanner_client_header_gen.process(xdg_shell_xml),
         ]
+        xdg_decoration_xml = wayland_protocols_datadir/'unstable/xdg-decoration/xdg-decoration-unstable-v1.xml'
+        xdg_decoration_sources = [
+            wayland_scanner_code_gen.process(xdg_decoration_xml),
+            wayland_scanner_client_header_gen.process(xdg_decoration_xml),
+        ]
```

Comment:
Fairly recent additions but just fyi: the [meson wayland module](https://mesonbuild.com/Wayland-module.html) is now stable so long term we should switch to using that.

Reply:
Good point, create #804 for long-term.

What is our policy related to min meson requirement? I have a pending patch to bump it to 1.1 (April 2023). I used a similar period (least 2 years old) than the previous bump.